.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         java-version: [1.8, 11, 15]
         # Tests seem to be failing on Windows
```

Comment:
can you remove this line and the next line?

Reply:
OMG, I didn't see it.
Done